apps/trading/components/accounts-container/accounts-container.tsx

```diff
@@ -75,3 +85,31 @@ export const AccountsContainer = ({
     </div>
   );
 };
+
+type Store = {
```

Comment:
Store shape for each table is the same. Do we need to defined it for each table separately?

```
const makeTableStateStore = (name: string) =>
  create<{
    gridStore: Store;
    update: (gridStore: Store) => void;
  }>()(
    persist(
      subscribeWithSelector((set) => ({
        gridStore: {},
        update: (newStore) => {
          set((curr) => ({
            gridStore: {
              ...curr.gridStore,
              ...newStore,
            },
          }));
        },
      })),
      { name }
    )
  );

const useLiquidityStore = makeTableStateStore('vega_ledger_store');
```